client/hardware.cpp

```diff
@@ -166,6 +171,7 @@ XrViewConfigurationView override_view(XrViewConfigurationView view, model m)
 		case model::htc_vive_xr_elite:
 			return scale_view(view, 1920);
 		case model::lynx_r1:
+		case model::galaxy_xr:
```

Comment:
If we follow the other device's conventions, this would be `scale_view(view, 3552)`.
I believe on 140% scale (the default) it would fail to create encoders and any computer would struggle to render.